runner/src/main.rs

```diff
@@ -73,10 +74,53 @@ fn build_examples(opt: &BuildExamples) -> Step {
     }
 }
 
+fn run_examples(opt: &RunExamples) -> Step {
+    match opt.application_variant.as_str() {
+        "rust" => Step::Multiple {
+            name: "examples".to_string(),
+            steps: vec![
```

Comment:
Having explicit metadata for each example rather than directory/file-naming conventions sounds good, but it would be better as some sort of manifest file that lives with the example &ndash; would that be the eventual aim?

Reply:
I like the manifest file idea. :)

Reply:
My main aim is to avoid inventing new programming or data definition languages, and do everything in Rust instead. So, the equivalent of a manifest will end up being expressed as actual Rust structs that can be manipulated and obey the rules of standard Rust.

Reply:
Totally fair. What I liked about David's manifest idea was that it lives in the dir of the example. 

Would it make sense to eventually have it stored there, potentially even in a rust file?

Not a biggie for me though, just seems nice to have it with the example.